Header header

Header image_header
sensor_msgs/Image image
int32 xres
int32 yres

object_detection/Lidar_Points lidar_points

int32 num_detects
Detection[] detections

int32 closest
